input_buffer.cc

```diff
@@ -337,6 +337,14 @@ input_buffer::consume(const char *str)
 	return false;
 }
 
+bool
+input_buffer::consume_char_literal(unsigned long long &outInt)
+{
+	outInt = (unsigned char)((*this)[0]);
```

Comment:
This currently won't work with escape sequences.  It would be nice to check if `outInt` is `'\'` and, if so, switch on the next character to handle `'\n'`, `'\r'`, `'\r'`, and `'\\'`, `'\''` and raise an error otherwise.

I believe the current code will error in the caller in this case.

Reply:
I didn't pay attention to such cases. I'll update